.eslintrc.json

```diff
@@ -1,10 +1,11 @@
 {
   "extends": "@callstack",
+  "parser": "@typescript-eslint/parser",
   "rules": {
     "global-require": 0,
     "flowtype/no-weak-types": 0,
     "strict": [0, "global"],
-    "no-unused-vars": ["error", { "argsIgnorePattern": "^_" }],
+    "@typescript-eslint/no-unused-vars": ["error", { "argsIgnorePattern": "^_" }],
```

Comment:
FYI we would get TS setup for free when upgrading callstack's eslint preset

Reply:
I assumed `no-unused-vars` wasn't in the base setup since it was added here

Reply:
Huh, I wanted to comment on the inclusion of `"parser": "@typescript-eslint/parser"` 😅

Reply:
yeah, worked fine without, now